locksmith/server.js

```diff
@@ -74,7 +75,7 @@ if (!config.host || env === 'production') {
 }
 
 // We wait for the db server to be up before starting the app
-serverIsUp(config.host, config.port, 100, 120, error => {
+serverIsUp(config.host, databasePort, 100, 120, error => {
```

Comment:
This adjustment is a bug fix to the service check script that manifested itself when running in the development environment